codalab/rest/interpret.py

```diff
@@ -300,17 +300,17 @@ def fetch_interpreted_worksheet(uuid):
     # The bundle_infos for bundles that don't need updating are also None.
     if bundle_uuids:
         for i, block in enumerate(interpreted_blocks['blocks']):
-            if 'bundle_info' not in block:
+            if not ('bundles_spec' in block and 'bundle_infos' in block['bundles_spec']):
```

Comment:
Can you explain a bit what your code changes do here? Maybe just in the PR description / a comment here

Reply:
It is used to check whether the block contains the field of 'bundle_infos' to determine whether it is a non-None block containing a list of bundles.